### ISSUE TITLE
Add VERSION property to the Hls object

### ISSUE THREAD

**Issue by shacharz** [videojs/videojs-contrib-hls#492]
Just like videojs has its version property (videojs.VERSION)
contrib-hls and other plugins should have one as well: videojs.Hls.VERSION

**Comment by dmlap** [comment]
Good idea. It seems like it would be a pretty easy PR for anyone interested in contributing.

**Comment by shacharz** [comment]
I might find time to do it, are you ok with setting a videojs.Hls.VERSION = '__VERSION__' somewhere in https://github.com/videojs/videojs-contrib-hls/blob/master/src/videojs-hls.js
and replacing it in grunt build, similar to https://github.com/videojs/video.js/blob/93f2f83723cee9cc229030b9586d1ca24a32d62f/build/grunt.js#L30 ? (just without browserify)

**Comment by dmlap** [comment]
We've swapped everything over to using babelify and npm scripts in the development branch. I don't want to add grunt back in just to do token replacement but I'd be down with a one-off script to do it.

**Comment by gkatsev** [comment]
there's a transform we're using in other places (like videojs) that can do a token replacement. https://www.npmjs.com/package/browserify-versionify
